src/patternfly/components/Button/examples/Button.md

```diff
@@ -406,6 +411,7 @@ Semantic buttons and links are important for usability as well as accessibility.
 | `.pf-m-warning` | `.pf-v5-c-button` | Modifies for warning styles. |
 | `.pf-m-link` | `.pf-v5-c-button` | Modifies for link styles. This button has no background or border and is styled as a link. This button would commonly appear in a form and may include an icon. |
 | `.pf-m-plain` | `.pf-v5-c-button` | Modifies for icon styles. This button has no background or border, uses a standard text color, and is used for `.pf-m-plain` icon buttons such as close, expand, kebab, etc. |
+| `.pf-m-no-padding` | `.pf-v5-c-button.pf-m-plain` | Modifies a plain button to remove padding. Should only be used when the button is inline within a sentence or block of text. Multiple plain buttons without padding should not be placed adjacent to one another without spacing between them. |
```

Comment:
I'd make the second sentence a full sentence - "This modifier should only be used when the button is inline within a sentence or block of text."
And then maybe simplify the last sentence? I don't love this wording but was trying to make it what TO do rather than what not to do.
"Adjacent plain buttons without padding should always have spacing between them."